src/common/unmarkedWsNoticeDetails/UnmarkedWsNoticeDetails.tsx

```diff
@@ -56,91 +55,29 @@ const UnmarkedWsNoticeDetails = ({
   boatModel,
   choice,
   summaryInformation,
+  leaseStatus,
+  leaseId,
 }: UnmarkedWsNoticeDetailsProps) => {
-  const { t, i18n } = useTranslation();
-
   return (
```

Comment:
`UnmarkedWsNoticeDetails` seems to have grown beyond a `common` component, since it now has queries and mutations inside it. It should probably be moved to `features`

Reply:
Moved to `features`.